components/locid_transform/src/provider.rs

```diff
@@ -135,6 +135,56 @@ pub struct AliasesV1<'data> {
     pub subdivision: ZeroMap<'data, UnvalidatedSubdivision, SemivalidatedSubdivision>,
 }
 
+// TODO: move to directionality module when stabilizing
+/// Represents the direction of a script.
+///
+/// [`LocaleDirectionality`] can be used to get this information.
+///
+/// <div class="stab unstable">
+/// 🚧 This code is considered unstable; it may change at any time, in breaking or non-breaking ways,
+/// including in SemVer minor releases. While the serde representation of data structs is guaranteed
+/// to be stable, their Rust representation might not be. Use with caution.
+/// </div>
+#[zerovec::make_ule(DirectionULE)]
```

Comment:
thought: should we organize htis crate further and put this in `provider/directionality.rs` or something?

Reply:
With the new RTL + LTR ZeroVec approach (https://github.com/unicode-org/icu4x/pull/3474#discussion_r1219178447), only the datastruct is in this file. The only other crate I was able to find that splits up the provider module is `icu_datetime`.

Reply:
Yeah I don't think this is necessary, the existing structs in this file are for different modules as well.